sdk/boms/azure-sdk-bom/pom.xml

```diff
@@ -179,7 +374,7 @@
               <reporters>html</reporters>
               <showAll>true</showAll>
               <analyseBom>true</analyseBom>
-              <failOnDependencyConflict>true</failOnDependencyConflict>
+              <!--<failOnDependencyConflict>true</failOnDependencyConflict>-->
```

Comment:
Why is this being commented out? If it needs to be removed then it should be removed entirely.

Reply:
Commented it out as we want to have this enabled but a few of the external dependencies conflict in their dependencies. I've filed a few issues against @JonathanGiles's plugin to add support for dependency inclusions/exclusions when running the report to get around them. They are trivial things that are second order dependencies that aren't a priority with what we want to present here.

Reply:
Removed the configuration.